manifests/reverse_proxy.pp

```diff
@@ -54,12 +54,12 @@
     ssl_options            => ['+StdEnvVars', '+ExportCertData', '+FakeBasicAuth'],
     ssl                    => true,
     ssl_proxyengine        => true,
-    ssl_proxy_ca_cert      => $certs::ca_cert,
+    ssl_proxy_ca_cert      => $certs::katello_default_ca_cert,
```

Comment:
Isn't this the certificate we expect on the remote server (so Foreman)? https://httpd.apache.org/docs/current/mod/mod_ssl.html#sslproxycacertificatefile states:
> This directive sets the all-in-one file where you can assemble the Certificates of Certification Authorities (CA) whose remote servers you deal with. These are used for Remote Server Authentication. Such a file is simply the concatenation of the various PEM-encoded Certificate files, in order of preference. This can be used alternatively and/or additionally to [SSLProxyCACertificatePath](https://httpd.apache.org/docs/current/mod/mod_ssl.html#sslproxycacertificatepath).

Reply:
Looking at https://github.com/theforeman/puppet-certs/commit/9585baa00b9158f8da16c36b76841df10b92fb97:
```
$ca_cert = "${pki_dir}/certs/${default_ca_name}.crt"
$katello_default_ca_cert = "${pki_dir}/certs/${default_ca_name}.crt"
```

so this is an equivalent change

Reply:
For this change,to fix tests I think we keep it equivalent. As for your point, this change dates back a long, long time to https://github.com/theforeman/puppet-foreman_proxy_content/pull/54.

The configuration is not setting SSLProxyVerify explicitly, where as we do set SSLClientVerify (granted to optional) so I am guessing this setting doesn't actually matter or do anything at the moment.